docs/hadrConfig.html

```diff
@@ -1,439 +1,882 @@
+      <br/>
+      <br/>
+      Furthermore, the database level parameter can be set to SYSTEM, meaning the database will use database manager level configuration. Otherwise, database level configuration overrides database manager level configuration.
+      <br/>
+      <br/>
+      The recommended value for HADR databases is RESTART. With this setting, at end of a takeover, the new primary database will search for all invalid indexes and rebuild them in background. Index rebuild does not prevent clients from connecting to the new primary database.
+      <br/>
+      <br/>
+      For more info, see
+      <a href="https://www.ibm.com/docs/en/db2/9.7?topic=parameters-indexrec-index-re-creation-time">
+        indexrec - Index re-creation time configuration parameter
+      </a>
+     </p>
+     <h2 dir="ltr">
+      <a name="DB2_LOAD_COPY_NO_OVERRIDE___Replaying_Load">
+       DB2_LOAD_COPY_NO_OVERRIDE - Replaying load
+      </a>
+     </h2>
+     <p dir="ltr">
+      HADR replicates load with certain restrictions. Because load data is not embedded in the log stream, the standby can only get the data from a load copy. Thus load is replicated only if there is a load copy. The copy must be available to the standby when it replays the load. The standby may attempt to access the copy any time after the primary completes the load. It is recommended that a shared copy device such as NFS is used. If you transfer the copy by means like physically transferring a tape, it is recommended that the standby be stopped ("db2 deactivated database") before the primary starts the load. Once the primary finishes the load and the copy is available to the standby, restart the standby ("db2 activate database"). The standby will then reconnect to the primary and replay the load.
+      <br/>
+      <br/>
+      When the primary does a load with COPY NO option, by default the load is automatically converted to NONRECOVERABLE. When standby replays the load, the table is marked as invalid. COPY NO loads can be optionally converted to COPY YES via the DB2_LOAD_COPY_NO_OVERRIDE registry variable. See
+      <a href="https://www.ibm.com/support/pages/load-operations-hadr">
+        Load operations and HADR in the Info Center page
+      </a>
+      for details.
+     </p>
+     <h2 dir="ltr">
+      <a name="BLOCKNONLOGGED___Block_Non_Logged_operations">
+       blocknonlogged - Block non logged operations
+      </a>
+     </h2>
+     <p dir="ltr">
+      First available in V95fp4 and V97.
+      <br/>
+      <br/>
+      When database parameter blocknonlogged is set to YES (default NO), tables with "not logged initially" attribute or not logged LOB columns can not created, either by create table or alter table statement. See also
+      <a href="https://www.ibm.com/docs/en/db2/9.7?topic=dcp-blocknonlogged-block-creation-tables-that-allow-non-logged-activity">
```

Comment:
Should use 11.5 version. In general, for definitions, we want to get links target latest version, unless we are specifically talking about an old version. 

There is another link below for autorestart, which needs to target 11.5 as well.